chef/cookbooks/pacemaker/files/default/reload_corosync_configuration.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# Script to re-load corosync configuration on a crowbar managed pacemaker
+# cluster.
+# This script can be used when you switch from single corosync ring to
+# dual corosync ring.
+# After you applied the modified proposal successfully, run this script on
+# crowbar
+# Please stop all chef-client runs while running this script.
+set -eux
+
+CLUSTER_PROPOSAL_NAME="$1"
+
+# Make sure that the proposal exists by asking for it's details
+# if this command fails, the script exits
+crowbarctl proposal show pacemaker "$CLUSTER_PROPOSAL_NAME"
+
+NODES=$(crowbarctl \
```

Comment:
Tiny nit (take it or leave it): stylistically, with multi-line `$( ... )` I prefer a line-break after the `$(` so that the arguments to `crowbarctl` are indented *more* than `crowbarctl`, not less, which also means that the commands in the pipeline are vertically aligned, and the `$( )` clearly delimit the whole pipeline:

```
NODES=$(
    crowbarctl proposal show pacemaker "$CLUSTER_PROPOSAL_NAME" \
        --format=plain \
        --filter "deployment.pacemaker.elements.pacemaker-cluster-member" |
    cut -d " " -f 2
)